public/app/plugins/panel/table/TablePanel.tsx

```diff
@@ -65,8 +97,8 @@ export class TablePanel extends Component<Props> {
 
     if (count > 1) {
```

Comment:
This means we're not rendering table for series with 1 row?
```suggestion
```

Reply:
there render at the end of function for the other case of only 1 data frame. We now render data frame selector at bottom if TablePanel gets more than 1 data frame , this case if for that